SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/DispatcherTest.cpp

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include <memory>
+
+#include "EntityComponentTestUtils.h"
+#include "SpatialView/Callbacks.h"
+#include "SpatialView/ComponentData.h"
+#include "SpatialView/Dispatcher.h"
+#include "SpatialView/EntityDelta.h"
+#include "SpatialView/EntityView.h"
+#include "Tests/TestDefinitions.h"
+
+#define DISPATCHER_TEST(TestName) GDK_TEST(Core, Dispatcher, TestName)
+
+namespace
+{
+constexpr Worker_ComponentId COMPONENT_ID = 1000;
+constexpr Worker_ComponentId OTHER_COMPONENT_ID = 1001;
+constexpr Worker_EntityId ENTITY_ID = 1;
+constexpr Worker_EntityId OTHER_ENTITY_ID = 2;
+constexpr double COMPONENT_VALUE = 3;
+constexpr double OTHER_COMPONENT_VALUE = 4;
+
+TUniquePtr<SpatialGDK::ComponentChange> CreateComponentAddedChange(const Worker_ComponentId ComponentId, const double Value)
```

Comment:
Why a unique pointer? This is a copyable object that doesn't own anything

Reply:
it was because the data inside is non-copyable, but I think what's actually happening is the pointer is copyable, so it's ok. Changed to be just the `ComponentChange` type rather than a unique pointer.